bitrise.yml

```diff
@@ -1,7 +1,6 @@
-format_version: 5
+format_version: "5"
 default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
-
-
+project_type: ""
 workflows:
   test:
     steps:
```

Comment:
Could you please update the go-tests for the newer solution
Example:
https://github.com/bitrise-io/go-xcode/blob/master/bitrise.yml#L17-L20

Reply:
I checked it but the same steps are used here too, the only difference is that in `go-list` a few `exclude`s are defined here.

Reply:
Yeah, you are right. The exclude list misled me for some reason